pkg/cosign/kubernetes/webhook/validator.go

```diff
@@ -411,15 +416,34 @@ func (v *Validator) resolvePodSpec(ctx context.Context, ps *corev1.PodSpec, opt
 	resolveContainers(ps.Containers)
 }
 
-func getFulcioCert(u *apis.URL) (*x509.CertPool, error) {
-	fClient := api.NewClient(u.URL())
-	rootCertResponse, err := fClient.RootCert()
+func getFulcioCert(fulcioURL *apis.URL) (*x509.CertPool, error) {
```

Comment:
Should we be providing this client in Fulcio to avoid duplication? I had hoped setting up a gRPC connection required very little boilerplate, but given that we need scheme/port detection, it might be good to centralize it.

fyi @wlynch, who would need the same in gitsign